File: .NET Core/Exceptions/InsufficientPrivilegesBackendException.cs
using com.contidio.sdk.proto;
using System.Net;

namespace Contidio.Sdk.Exceptions
{
    public class InsufficientPrivilegesBackendException : BackendException
    {
        public InsufficientPrivilegesBackendException()
            : base(BackendErrorCode.INSUFFICIENT_PRIVILEGES, "Insufficient privileges", "InsufficientPrivilegesBackendException", HttpStatusCode.Forbidden)
        {
        }
    }
}

File: Java/src/main/java/com/contidio/sdk/exceptions/InsufficientPrivilegesBackendException.java
package com.contidio.sdk.exceptions;

import org.apache.commons.httpclient.HttpStatus;

import com.contidio.sdk.proto.MetadataProtocol.BackendErrorCode;

@SuppressWarnings("serial")
public class InsufficientPrivilegesBackendException extends BackendException {
	public InsufficientPrivilegesBackendException() {
		super(BackendErrorCode.INSUFFICIENT_PRIVILEGES, "Insufficient privileges", "InsufficientPrivilegesBackendException", 
				HttpStatus.SC_FORBIDDEN);
	}
}
